canonical-web-and-design/react-components
# Fix docs props

Done:
- Fixed props that weren't appear in the static docs. Fixes: https://github.com/canonical-web-and-design/react-components/issues/4.
- Rebuilt the docs.

QA:
- pull this branch and open `docs/index.html` in a browser. The props block for the components should appear.
